apps/web-giddh/src/assets/css/style-1.scss

```diff
@@ -1271,6 +1271,27 @@ table:focus {
 .navbar-right.navbar-nav > li > a {
     border-right: 0;
 }
+.all-table {
+    .mat-header-cell {
+        padding: 0px 6px !important;
+        height: 51.5px;
+        &:last-child {
+            padding-left: 20px !important;
+        }
+    }
+    tbody {
+        td {
```

Comment:
Can we create mixin for this as it is used at different places

Reply:
It will be not applied because because there in media query you have to define at wihout media  so it is applying to media queries but there is also define in same class in top